Applications/opensense/opensense.cpp

```diff
@@ -103,6 +108,11 @@ int main(int argc, char **argv)
                     return 0;
                 }
                 else if ((option == "-ReadA") || (option == "-RA")) {
+                    if (argc < 4) {
+                        cout << "Both data file (.csv) and setup file needed to read APDM data. Please fix and retry." << endl;
```

Comment:
`"Both the data file (.csv) with APDM formatted data and the reader settings file are necessary to read APDM data. Please retry with these inputs."`